cmd/skaffold/app/cmd/flags.go

```diff
@@ -384,6 +384,14 @@ var flagRegistry = []Flag{
 		FlagAddMethod: "StringVar",
 		DefinedOn:     []string{"build", "debug", "dev", "run", "deploy"},
 	},
+	{
+		Name:          "platform",
+		Usage:         "The platform to target for the build artifacts",
```

Comment:
should we point to docs here to give the user more information about accepted values?